libsolidity/formal/Why3Translator.cpp

```diff
@@ -757,6 +757,11 @@ bool Why3Translator::visit(Literal const& _literal)
 	return false;
 }
 
+bool Why3Translator::visit(PragmaDirective const&)
+{
+	return false;
```

Comment:
I changed the code to use the SemVerParser to detect a version.  However, the parse accepts too many strings.  Currently most malformed pragmas are parsed as version pragmas; see #1192.